### PR TITLE
modify entry using put just failing

### PR DESCRIPTION
**Pull request by Janet-Namutebi**
Hi @dennisja, when I try to update an entry, it throws back my written except message. I think it's because of the way am trying to implement that put method but I can't figure it out.
```
janetblessed@janetblessed-OptiPlex-3010:~/git/environments/diary_api_$ curl -i -H "Content-Type: application/json" -X PUT -d '{"title":"recursive","content":"recursion with power"}' http://localhost:5004/api/v1/entries/3
HTTP/1.0 401 UNAUTHORIZED
Content-Type: application/json
Content-Length: 37
Server: Werkzeug/0.14.1 Python/3.5.2
Date: Tue, 25 Sep 2018 09:56:42 GMT

{
  "message": "JSON Format Error"
}
```

### CODE REVIEW

**Comment by Janet-Namutebi** [comment]
It a nice idea though, I will use it in the next project an intending to do

**Comment by dennisja** [comment]
No
In this

File: app.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                "title": title,
+                "content":content,
+                "time":get_timestamp()
+                }
+
+            entries.append(entrycontent)
+
+            return make_response(jsonify(
+                {'entrycontent': entrycontent},
+                {'message': "Entry successfully added"}), 200)
```
Comment (dennisja) [inline app.py]: This is a bad status code

File: app.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            entry[0]['content'] = content
+            entry[0]['time'] = get_timestamp()
+            
+            return make_response(jsonify(
+                {'entry':entry[0]},
+                {'message': "Entry successfully updated"}), 201)
+                
+        except (ValueError, KeyError, TypeError):
+            return make_response(jsonify(
+                {'message': "JSON Format Error"}), 401)
```
Comment (dennisja) [inline app.py]: This status code is also bad

**Comment by Janet-Namutebi** [comment]
Hi @dennisja , am having issues with my tests. Actually this error.
```
E
======================================================================
ERROR: test_add_entry_successfully_with_post (__main__.TestEndpoint)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_entry.py", line 56, in test_add_entry_successfully_with_post
    data = json.load(reader(response.data))
  File "/usr/lib/python3.5/json/__init__.py", line 265, in load
    return loads(fp.read(),
  File "/usr/lib/python3.5/codecs.py", line 493, in read
    newdata = self.stream.read()
AttributeError: 'bytes' object has no attribute 'read'

----------------------------------------------------------------------
Ran 1 test in 0.003s

FAILED (errors=1)
```

**Pull request merged**